pkg/devfile/adapters/docker/component/utils.go

```diff
@@ -256,7 +219,7 @@ func (a Adapter) startComponent(mounts []mount.Mount, projectVolumeName string,
 	// Create the docker container
 	s := log.Spinner("Starting container for " + *comp.Image)
 	defer s.End(false)
-	err = a.Client.StartContainer(&containerConfig, &hostConfig, nil)
+	_, err = a.Client.StartContainer(&containerConfig, &hostConfig, nil)
```

Comment:
Nope, we just return..

We only had to update this func for container id, so that we removed the supervisord init container after starting it, so that it doesn't clutter up `docker ps -a`